Adding table shortcode for multi-line tables (#17578)
yugabyte/yugabyte-db: docs/content/preview/contribute/docs/docs-style.md

#### YSQL and YCQL code blocks

> Tag YSQL code blocks as `plpgsql`, and YCQL code blocks as `cql`. The source highlighting differs slightly between the two.

Reviewer: what's the reason for tagging YSQL code blocks as `plpgsql`?

Author: `plpgsql` has features like `tablespaces` and other postgres specific features that `sql` does not have . so they will be highlighted too.

> ### Tables

Reviewer 2: Content more accurately goes on page https://docs.yugabyte.com/preview/contribute/docs/widgets-and-shortcodes/

Author: Let me move this to that section.

> Markdown supports [tables](https://www.markdownguide.org/extended-syntax/#tables). By design the table rows have to be in one line. This makes adding bullets and multi-line code blocks impossible in table cells. For this, use the shortcode `table`. For example,

Reviewer: ```suggestion
Markdown supports [tables](https://www.markdownguide.org/extended-syntax/#tables). By design, the table rows have to be in one line making it impossible to add bullets and multi-line code blocks in table cells. To create such custom tables, use the shortcode `table` similar to the following example:
```

```md
{{</*table*/>}}
| col-1 | col-2 |
| ----- | ----- |
<!-- row with bullets and code block -->
|
- 1
- 2

> - 3|

Reviewer 2: ```suggestion
- 3 |
```

```output
 k | v
---+---
 1 | 2
(1 row)
```|
<!-- row with tip block -->
| {{</*warning title="Beware" */>}} start and end rows with the pipe symbol {{</*/warning*/>}} | 
{{</*tip title="Awesome tip" */>}} Use 3 ticks for code blocks with pipe symbols {{</*/tip*/>}} |
{{</*/table*/>}}
```

> The above markdown should render as,

Reviewer: ```suggestion
The above markdown should render a table as follows:
```
